site/snippets/react/useBundlerClient.tsx

```diff
@@ -0,0 +1,21 @@
+import { useBundlerClient } from "@alchemy/aa-alchemy/react";
+import { useQuery } from "@tanstack/react-query";
+
+export function ComponentWithBundlerClient() {
+  const client = useBundlerClient();
+
+  const { data, isLoading } = useQuery({
+    queryKey: ["supportedEntryPoints"],
+    queryFn: () => client.getSupportedEntryPoints(),
+  });
```

Comment:
maybe this should be a hook too? could also leverage the refactor since bundler client is on the external store

Reply:
yea we probably could create hooks for the bundler client methods as well, but I think they're probs less impactful than the methods on `SmartAccountClient`